tools/node_modules/terser/terser.js

```diff
@@ -1608,6 +1608,10 @@ TreeWalker.prototype = {
         }
     }
 };
+
+// XXX Emscripten: export TreeWalker for walking through AST in acorn-optimizer.js.
```

Comment:
Should we open a PR upstream?

Especially since the plan is to move to using `npm` to install stuff like this.

Reply:
Upstream has that export fixed - but I was not able to update because upstream regresses `keep_quoted_props` (https://github.com/emscripten-core/emscripten/pull/10272/files#diff-878a436346e868f70b712efa5512f7b0R976). In latest terser that option no longer works, and that broke our use of Closure.